src/appimagetool.c

```diff
@@ -882,6 +882,9 @@ main (int argc, char *argv[])
         if (verbose)
             printf("Size of the embedded runtime: %d bytes\n", size);
         
+        if (verbose)
```

Comment:
This change is completely unrelated. You should either move your commit to a new branch and send a separate PR or clean up your Git history so we can merge it as a part of this PR.

... also, I'm not sure whether it's redundant to the other messages.